staging/vhost-device-gpu/src/virtio_gpu.rs

```diff
@@ -245,6 +245,12 @@ pub trait VirtioGpu {
         context_init: u32,
         context_name: Option<&str>,
     ) -> VirtioGpuResult;
+
+    /// Get an EventFd descriptor, that signals when to call event_poll.
+    fn get_event_poll_fd(&self) -> Option<EventFd>;
+
+    /// Polls events.
```

Comment:
```suggestion
    /// Polls the Rutabaga backend.
```
not very descriptive I will suggest same as crosvm.